main/src/ca/uwaterloo/flix/api/lsp/provider/completion/ExprCompleter.scala

```diff
@@ -23,7 +23,7 @@ import ca.uwaterloo.flix.language.ast.TypedAst
 object ExprCompleter {
 
   def getCompletions(context: CompletionContext)(implicit flix: Flix, index: Index, root: TypedAst.Root): Iterable[Completion] = {
-    DefCompleter.getCompletions(context) ++
+//    DefCompleter.getCompletions(context) ++
```

Comment:
The duty of this part will later be taken by the `Declaration` part in `LocalScopeCompletion`.

When it's not in scope, `AutoUseCompletion` will provide the choice. Otherwise, `LocalScopeCompletion` will take it.

This is also part of moving SyntacticCompletion to SemanticCompletion.